website/projects.html

```diff
@@ -20,6 +20,7 @@
 		<!-- Page links -->
 		<div class="topnav-right">
 			<div class="topnav-placeholder"></div>
+			<a class="topnav-link" href="login.html">Login</a>
```

Comment:
Good thinking adding a link to the topnav (I forgot it in the original issue haha).